src/blocks/Slider/Slider.tsx

```diff
@@ -30,8 +30,9 @@ import {
     getSlidesToShowWithDefaults,
 } from './utils';
 
-import './Slider.scss';
 import './slick.scss';
+// eslint-disable-next-line import/order
```

Comment:
can we add the reason of disabling in comment?

Reply:
Yep, add reason